sdk-lib/src/main/java/cash/z/ecc/android/sdk/internal/db/derived/AllTransactionView.kt

```diff
@@ -34,7 +34,7 @@ internal class AllTransactionView(
             Locale.ROOT,
             "%s DESC, %s DESC", // $NON-NLS
             COLUMN_SORT_HEIGHT,
-            AllTransactionViewDefinition.COLUMN_INTEGER_ID
+            AllTransactionViewDefinition.COLUMN_INTEGER_TRANSACTION_INDEX
```

Comment:
This change means we sort transactions in "block height, followed by order within block" order (which is mostly how it would appear previously, except that transactions we created in this wallet will show in a slightly later spot if they are created before we receive some other tx, but mined after).